moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js

```diff
@@ -73,13 +73,34 @@ const useColumns = ({
           return user ? `${user.first_name} ${user.last_name}` : "";
         },
         renderEditCell: (props) => {
+          // the team member object for the current row
+          const currentRowMember =
+            data?.moped_project_by_pk?.moped_proj_personnel.filter(
```

Comment:
small thing but if you use `.find()` here then you will return the object instead of the object wrapped in an array.

Reply:
ooo til, thank u!